tests/monitors/collectd_mysql/mysql_test.py

```diff
@@ -0,0 +1,97 @@
+import time
+from functools import partial as p
+
+import mysql.connector
+
+import pytest
+from tests.helpers.agent import Agent
+from tests.helpers.assertions import has_datapoint_with_dim, has_datapoint_with_metric_name, tcp_socket_open
+from tests.helpers.metadata import Metadata
+from tests.helpers.util import container_ip, ensure_always, run_container, wait_for
+from tests.helpers.verify import (
+    verify,
+    verify_expected_is_subset,
+    run_agent_verify_included_metrics,
+    run_agent_verify_all_metrics,
+)
+
+pytestmark = [pytest.mark.docker_container_stats, pytest.mark.monitor_without_endpoints]
+
+ENV = {
+    "MYSQL_DATABASE": "testdb",
+    "MYSQL_USER": "testuser",
+    "MYSQL_PASSWORD": "testpass",
+    "MYSQL_ROOT_PASSWORD": "testpass",
+}
+
+METADATA = Metadata.from_package("collectd/mysql")
+
+
+def test_mysql_included():
+    with run_container("mysql:5.7", environment=ENV) as mysql:
+        host_ip = container_ip(mysql)
+        assert wait_for(p(tcp_socket_open, host_ip, 3306), 60), "service didn't start"
+
+        create_db_activity(host_ip)
+
+        run_agent_verify_included_metrics(
+            f"""
+            monitors:
+            - type: collectd/mysql
+              host: {host_ip}
+              port: 3306
+              username: {ENV.get("MYSQL_USER")}
```

Comment:
I think `ENV["MYSQL_USER"]` would be better to prevent silently putting `None` in there if you always expect those values to be set in `ENV`.

Reply:
Another option would be to do
```
"""
username: {MYSQL_USER}
...
""".format(**ENV)
```